### PR TITLE
Create lepao.md

### PR DESCRIPTION
**Pull request by luis-gonzalez-m**
As per indicated in https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1669#issuecomment-993803601
@matentzn

### CODE REVIEW

File: ontology/lepao.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: ontology_detail
+id: lepao
+mirror_from: https://raw.githubusercontent.com/insect-morphology/lepao/main/lepao.owl
+title: Lepidoptera Anatomy Ontology
+contact:
+  email: lagonzalezmo@unal.edu.co; luis.gonzalez.montana@unillanos.edu.co
```
Comment (cthoyt) [inline ontology/lepao.md:7]: This isn’t a single point of contact

Comment (matentzn) [reply ontology/lepao.md]: Changed it

**Comment by matentzn** [comment]
@luis-gonzalez-m I added your github handle and changed your contact email. Can you make sure its correct now? There can only be one contact email.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
I am assuming its fine; please make another pull request if not

**Pull request merged**